server.js

```diff
@@ -6159,6 +6160,13 @@ function escapeFormat(t) {
     .replace(/__/g, '_').replace(/--/g, '-');
 }
 
+function escapeFormatSlashes(t) {
+  return escapeFormat(t)
+    // Double slash
+    .replace(/\/\//g, '//');
+}
```

Comment:
#952 does not seem to move the definition of `escapeFormat`, so I suppose this should be okay, right?